kuma/wiki/tests/test_jobs.py

```diff
@@ -52,5 +54,6 @@ def test_contributors(db, cleared_cacheback_cache, settings, wiki_user_3,
 
     # The new contributor shows up and is first, followed
     # by the freshly un-banned user, and then the rest.
+    contributors = job.get(root_doc.pk)
```

Comment:
Just noticed that this is unused. I suspect you forgot to use it on the line below in place of `job.get(root_doc.pk)`?